postgres2.yml

```diff
@@ -1,106 +1,78 @@
-ttl: &ttl 30
-loop_wait: &loop_wait 10
-scope: &scope batman
+scope: batman
+#namespace: /service/
+name: postgresql2
+
 restapi:
   listen: 127.0.0.1:8010
   connect_address: 127.0.0.1:8010
-  auth: 'username:password'
 #  certfile: /etc/ssl/certs/ssl-cert-snakeoil.pem
 #  keyfile: /etc/ssl/private/ssl-cert-snakeoil.key
+  authentication:
```

Comment:
Well, it only works because we are not issuing API requests to postgres2 from the acceptance tests. Anyone extending those tests to check the status of those instance (i.e. adding tag-related tests) will be unpleasantly surprised.